crates/nu-cli/src/nu_highlight.rs

```diff
@@ -14,6 +16,11 @@ impl Command for NuHighlight {
     fn signature(&self) -> Signature {
         Signature::build("nu-highlight")
             .category(Category::Strings)
+            .switch(
+                "reject-garbage",
+                "Return an error if invalid syntax (garbage) was encountered",
+                None,
```

Comment:
Is there a reason this switch doesn't have a short like `Some('r')` or `Some('g')`?

Reply:
i figured it would be rare enough you would want to spell it out all the way but i'm fine with adding a short flag too

Reply:
We can always add one later if we need to. I generally like to have long and short but it's no big deal.